openedx/core/lib/api/authentication.py

```diff
@@ -54,19 +54,13 @@ def authenticate(self, request):
 
         if auth and auth[0].lower() == b'bearer':
             access_token = auth[1].decode('utf8')
-            set_custom_metric('OAuth2Authentication_token_location', 'bearer-in-header')
-        elif 'access_token' in request.POST:
-            access_token = request.POST['access_token']
-            set_custom_metric('OAuth2Authentication_token_location', 'post-token')
```

Comment:
Does this mean `post-token` was never seen?

Reply:
yes, it is never seen in metric data